pkg/eraser/eraser.go

```diff
@@ -22,9 +22,48 @@ const (
 
 var (
 	// Timeout  of connecting to server (default: 10s)
-	timeout time.Duration = 100 * time.Second
+	timeout = 10 * time.Second
 )
 
+type client struct {
+	images  pb.ImageServiceClient
+	runtime pb.RuntimeServiceClient
+}
+
+func (c *client) listContainers(context.Context) (list []*pb.Container, err error) {
+	resp, err := c.runtime.ListContainers(context.Background(), new(pb.ListContainersRequest))
+	if err != nil {
+		return nil, err
+	}
+	return resp.Containers, nil
+}
+
+func (c *client) listImages(ctx context.Context, image string) (list []*pb.Image, err error) {
```

Comment:
At a glance, it seems like we always pass an empty string here for `image`, do we need the filter?